### PR TITLE
BP-337: Frontend: location item add/update/delete page

### PR DESCRIPTION
**Pull request by stefashkaa**
<h1>Description</h1>
<p>I've added project. Also, I've developed merchant info page. Moreover, I've implemented add/update merchant dialog.</p>

https://bakong.atlassian.net/browse/BP-335
https://bakong.atlassian.net/browse/BP-337

<h1>Works done</h1>

- [x] Add project
- [x] Develop merchant info page
- [x]  Implement add/update merchant dialog

### CODE REVIEW

File: src/views/MerchantDetails/MerchantDetails.jsx [hunk 1]
```diff
@@ -0,0 +1,325 @@
...
+                  </FormGroup>
+                  <FormGroup>
+                    <Row>
+                      <Col md="3">
+                        <Label>Coordinates:</Label>
+                      </Col>
+                      {coordinates && (
+                        <Col xs="12" md="9">
+                          <Label>
+                            latitude - {coordinates.lat}, longitude -{" "}
```
Comment (Asmadek) [inline src/views/MerchantDetails/MerchantDetails.jsx:195]: Eslint make it horrible, maybe divide it like this
```
latitude - {coordinates.lat},
longitude - {coordinates.lng}
```

File: public/index.html [hunk 2]
```diff
@@ -0,0 +1,78 @@
...
+  <head>
+    <meta charset="utf-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta
+      name="viewport"
+      content="width=device-width, initial-scale=1, shrink-to-fit=no"
+    />
+    <meta
+      name="description"
+      content="BAKONG using CoreUI for React - Open Source Bootstrap Admin Template"
```
[the remark this answers is not in the document]
Comment (stefashkaa) [reply public/index.html:12]: Ok

File: public/manifest.json [hunk 3]
```diff
@@ -0,0 +1,15 @@
+{
+  "short_name": "BAKONG Web App",
```
[the remark this answers is not in the document]
Comment (stefashkaa) [reply public/manifest.json:2]: Ok

File: Jenkinsfile [hunk 4]
```diff
@@ -0,0 +1,2 @@
+@Library('jenkins-library' ) _
+new org.bakong.frontendLibrary().call(nexusPushImageName:'nbcfast/local-frontend',e2eTest: true)
```
[the remark this answers is not in the document]
Comment (stefashkaa) [reply Jenkinsfile:2]: Ok, I'll comment it

File: src/views/MerchantDetails/MerchantDetails.jsx (hunk 1 above)
Comment (stefashkaa) [reply src/views/MerchantDetails/MerchantDetails.jsx:195]: Ok

**Pull request merged**